src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -1142,9 +1146,13 @@ namespace RuntimeLanguageProcessor {
 		} else if ( String[ 0 ] == '+' ) {
 			String = "0" + String;
 		}
-		std::string::size_type const LastPos( String.length() );
+		std::string::size_type LastPos( String.length() );
 		Pos = 0;
-		while ( Pos < LastPos ) {
+		OperatorProcessing = false; // true when an operator is found until terminated by non-operator
+		MinusFound = false;
+		MultFound = false;
+		DivFound = false;
```

Comment:
Th OperatorProcessing and MinusFound had to be moved up so as not to reset them. Same for the new variables to catch bad equation format.